.zenodo.json

```diff
@@ -1,126 +1,258 @@
-      },
-      {
-        "affiliation": "HZDR",
-        "name": "Göthel, Ilja"
-      },
-      {
-        "affiliation": "HZDR, TU Dresden",
-        "name": "Rudat, Sophie"
+  "creators":[
+    {
+      "affiliation":"HZDR",
+      "name":"Bastrakov, Sergei",
+      "orcid":"0000-0003-3396-6154"
+    },
+    {
+      "affiliation":"HZDR",
+      "name":"Bastrakova, Kseniia"
+    },
+    {
+      "name":"Marre, Brian Edward",
+      "affiliation":"HZDR, TU Dresden"
+    },
+    {
+      "affiliation":"HZDR",
+      "name":"Debus, Alexander",
+      "orcid":"0000-0002-3844-3697"
+    },
+    {
+      "affiliation":"HZDR, TU Dresden",
+      "name":"Garten, Marco",
+      "orcid":"0000-0001-6994-2475"
+    },
+    {
+      "name":"Gruber, Bernhard Manfred",
+      "affiliation":"CASUS, HZDR, CERN",
+      "orcid":"0000-0001-7848-1690"
+    },
+    {
+      "affiliation":"HZDR, TU Dresden, LBNL",
+      "name":"Huebl, Axel",
+      "orcid":"0000-0003-1943-7141"
+    },
+    {
+      "name":"Trojok, Jakob",
+      "affiliation":"HZDR, TU Dresden"
+    },
+    {
+      "affiliation":"HZDR",
+      "name":"Kelling, Jeffrey",
+      "orcid":"0000-0003-1761-2591"
+    },
+    {
+      "affiliation":"HZDR, TU Dresden",
+      "name":"Lebedev, Anton"
+    },
+    {
+      "name":"Meyer, Felix",
+      "affiliation":"HZDR"
+    },
+    {
+      "name":"Ordyna, Paweł",
+      "affiliation":"HZDR, TU Dresden"
+    },
+    {
+      "name":"Poeschel, Franz",
+      "affiliation":"CASUS, HZDR",
+      "orcid":"0000-0001-7042-5088"
+    },
+    {
+      "name":"Sprenger, Lennert",
+      "affiliation":"HZDR, TU Dresden"
+    },
+    {
+      "affiliation":"HZDR",
+      "name":"Steiniger, Klaus",
+      "orcid":"0000-0001-8965-1149"
+    },
+    {
+      "name":"Wang, Manhui",
+      "affiliation":"University of Liverpool"
+    },
+    {
+      "affiliation":"HZDR",
+      "name":"Starke, Sebastian"
+    },
+    {
+      "name":"Thévenet, Maxence",
+      "affiliation":"Desy"
+    },
+    {
+      "affiliation":"HZDR, TU Dresden",
+      "name":"Pausch, Richard",
+      "orcid":"0000-0001-7990-9564"
+    },
+    {
+      "affiliation":"HZDR",
+      "name":"Widera, René",
+      "orcid":"0000-0003-1642-0459"
+    }
+  ],
+  "contributors":[
+    {
+      "affiliation":"LOA",
+      "name":"Andriyash, Igor",
+      "type":"Other"
+    },
+    {
+      "affiliation":"University Jena",
+      "name":"Bifeng, Lei",
+      "type":"Other"
+    },
+    {
+      "affiliation":"ELI-NP",
+      "name":"Berceanu, Andrei",
+      "type":"Other"
+    },
+    {
+      "affiliation":"HZDR, TU Dresden",
+      "name":"Burau, Heiko",
+      "type":"Other"
+    },
+    {
+      "affiliation":"HZDR",
+      "name":"Bussmann, Michael",
+      "orcid":"0000-0002-8258-3881",
+      "type":"Other"
+    },
+    {
+      "affiliation":"HZDR, TU-Dresden",
+      "name":"Carstens, Finn-Ole",
+      "type":"Other"
+    },
+    {
+      "affiliation":"HZDR, TU-Dresden",
```

Comment:
outdated?

Reply:
No this is fine, he was TUD student and HZDR